doc/recording_vcr_tests.md

```diff
@@ -1,105 +1,64 @@
 
-To create a new test, simply create a class in the `test_<module>_commands.py` file with the following structure:
-
-```Python
-
-class MyTestClass(ResourceGroupVCRTestBase): # or VCRTestBase in special circumstances
-
-  def __init__(self, test_method):
-    # TODO: replace MyTestClass with your class name
-    super(MyTestClass, self).__init__(__file__, test_method, debug=False, run_live=False, skip_setup=False, skip_teardown=False)
-      
-  def test_my_test_class(self): # TODO: rename to 'test_<your name here>'
-    self.execute()
-
-  def body(self):
-    # TODO: insert your test logic here
-    
-  def set_up(self):
-    super(MyTestClass, self).set_up() # if you need custom logic, be sure to call the base class version first
-    # TODO: Optional setup logic here (will not be replayed on playback)
-    
-    
-  def tear_down(self):
-    # TODO: Optional tear down logic here (will not be replayed on playback)
-    super(MyTestClass, self).tear_down() # if you need custom logic, call the base class version last
-```
+The Azure CLI relies on [VCR.py](https://vcrpy.readthedocs.io/en/latest/) to record and replay HTTP communications. On top of the VCR.py, we build [ScenarioTest](scenario_base_test.md) to facilitate authoring tests.
 
-The `debug`, `run_live`, `skip_setup` and `skip_teardown` parameters in the `__init__` method are shown with their defaults and can be omitted. `debug` is the equivalent of specifying `debug=True` for all calls to `cmd` in the test (see below). Specifying `run_live=True` will cause the test to always be run with actual HTTP requests, ignoring VCR entirely. `skip_setup` and `skip_teardown` can be useful during test creation to avoid repeatedly creating and deleting resource groups.
+## Authoring Tests
 
-The `set_up` and `tear_down` methods are optional and can be omitted. For the ResourceGroupVCRTestBase these have default implementations which set up a test resource group and tear it down after the recording completes. Any commands used in these methods are only executed during a live or recorded test. These sections are skipped during playback so your test body should not rely any logic within these methods.
+https://github.com/Azure/azure-cli/blob/master/doc/scenario_base_tests.md
 
-A number of helper methods are available for structuring your script tests.
+### Tips
 
-#### cmd(command_string, checks=None, allowed_exceptions=None, debug=False)
+* Make the tests delete the Azure resources after execution.
+* Name the test method in following format: `test_<module>_<feature>`.
 
-This method executes a given command and returns the output. If the output is in JSON format, the method will return the results as a JSON object for easier manuipulation.
```

Comment:
You should call out "get_output_as_json" since this is no longer the default behavior.